app/models/agents/user_location_agent.rb

```diff
@@ -14,6 +14,8 @@ class UserLocationAgent < Agent
       If you want to only keep more precise locations, set `max_accuracy` to the upper bound, in meters. The default name for this field is `accuracy`, but you can change this by setting a value for `accuracy_field`.
 
       If you want to require a certain distance traveled, set `min_distance` to the minimum distance, in meters. Note that GPS readings and the measurement itself aren't exact, so don't rely on this for precision filtering.
+
+      To view the locations on a map, set `api_key` to your Google Maps JavaScript API key.
```

Comment:
Yes, good idea.

> On 03.08.2016, at 16:45, Andrew Cantino <notifications@github.com> wrote:
> 
> In app/models/agents/user_location_agent.rb:
> 
> > @@ -14,6 +14,8 @@ class UserLocationAgent < Agent
> >        If you want to only keep more precise locations, set `max_accuracy` to the upper bound, in meters. The default name for this field is `accuracy`, but you can change this by setting a value for `accuracy_field`.
> >  
> >        If you want to require a certain distance traveled, set `min_distance` to the minimum distance, in meters. Note that GPS readings and the measurement itself aren't exact, so don't rely on this for precision filtering.
> > +
> > +      To view the locations on a map, set `api_key` to your Google Maps JavaScript API key.
> Maybe link this to https://developers.google.com/maps/documentation/javascript/get-api-key#key ?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>